devtools-ci-index.yaml

```diff
@@ -4542,6 +4542,28 @@
             git_repo: f8a-stacks-report
         - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
             git_repo: f8a-stacks-report
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8-analytics
+            git_repo: fabric8-analytics-bigquery-manifests-job
+            saas_git: saas-analytics
+            prj_name: bayesian-preview
+            saas_service_name: f8a-bq-manifests-job
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
```

Comment:
this file does not exist in https://github.com/fabric8-analytics/fabric8-analytics-bigquery-manifests-job

Reply:
Hey @jmelis yes I'm supposed to push those files later on today, should I rebase and ping you once I add these files?

Reply:
added this now